chapter-06/Exercise_01/Conversoes.scala

```diff
@@ -0,0 +1,12 @@
+/* 01. Write an object Conversions with methods inchesToCentimeters , gallonsToLiters , and
+milesToKilometers .*/
+
+
+package Exercise_01
+
+object Conversoes extends App {
+  def InchesToCentimeters(inches: Double) = inches * 2.54
```

Comment:
Não se esqueça de que os nomes de métodos devem seguir o padrão `kebabCase`:
```suggestion
  def inchesToCentimeters(inches: Double) = inches * 2.54
```

Reply:
ok!